exercises/twofer/src/lib.rs

```diff
@@ -0,0 +1,3 @@
+pub fn twofer(_name: &str)-> String {
```

Comment:
Instead of using an underscore prefix for the variable, try to fit it into the `unimplemented` statement. It's fine to get cute with it: `unimplemented!("One for {}, one for me.", name)`.